indexer/docker-compose.yml

```diff
@@ -75,5 +75,121 @@ services:
       postgres:
         condition: service_healthy
 
+  gateway-frontend:
```

Comment:
note: this docker compose has never worked (prs to make it work never landed and we don't use docker-compose locally or in e2e tests atm).

Side note: would be good to use docker-compose on circleci instead of manually setting up postgres since it makes running the tests way easier and 1:1 with CI. Not important will do this change if it ever becomes useful enough to do

In future I will make this entire thing work e2e via building the gateway locally making all the changes the gateway needs and here to get everything running. This just sets up being able to start that process